Forte.EpiServer.AzureSearch/Events/SearchEventHandler.cs

```diff
@@ -91,8 +91,13 @@ private static bool IsContentMarkedAsExpired(ContentEventArgs contentEventArgs,
 
         private PageData GetPagePreviousVersion(ContentReference reference)
         {
+            if (ContentReference.IsNullOrEmpty(reference))
+            {
+                return null;
+            }
+            
             var previousPageReference = reference.ToReferenceWithoutVersion();
-            return _contentLoader.Get<PageData>(previousPageReference);
+            return previousPageReference != null ? _contentLoader.Get<PageData>(previousPageReference) : null;
```

Comment:
Do we need this null check? I mean if we're already sure that reference is not null is there a scenario in which `ToReferenceWithoutVersion` will still return null?